english/open-door/open-door.md

```diff
@@ -6,3 +6,7 @@ outside the room you see...
 
 That you are about to be crushed by a giant corn. :)
 
+     or 
+
+[The nose wheel light of a 747 that is landing directly in front of you](../airplane-princess/slide.md)
```

Comment:
I understand.  Thank you!   Paul


> On Jul 28, 2015, at 12:11 PM, Caroline Buckey <notifications@github.com> wrote:
> 
> In english/open-door/open-door.md:
> 
> > @@ -6,3 +6,7 @@ outside the room you see...
> >  
> >  That you are about to be crushed by a giant corn. :)
> >  
> > +     or 
> > +
> > +[The nose wheel light of a 747 that is landing directly in front of you](../airplane-princess/slide.md)
> The .. is not necessary, since airplane-princess is in the same directory as this file.
> 
> —
> Reply to this email directly or view it on GitHub.
>